### ISSUE TITLE
Feature Request: Debugger colour theme

### ISSUE THREAD

**Issue by robert-yates** [Vector35/debugger#451]
I'd like the ability to have an alternative colour theme applied once a debugger view starts.

**Comment by xusheng6** [comment]
Does the built-in theme support for BN work for your case?

**Comment by robert-yates** [comment]
yeah should be ok, i could take my current theme and then make a debugger version, just need to somehow trigger the switching

**Comment by xusheng6** [comment]
Oh you want the switching! It could be done be emitting a theme change event when we launch the target, though this is gonna have a fairly low priority for now

**Comment by robert-yates** [comment]
ah yes sorry if that wasnt clear, i want the starting of the debugger to switch to my debugger only theme and then switch back after the debugger session ends.

yeah a theme change event would be perfect, but i understand it will be low priority, thanks !
/R.
